src/lib/parameters/flashparams/flashparams.cpp

```diff
@@ -265,7 +250,7 @@ param_import_callback(bson_decoder_t decoder, void *priv, bson_node_t node)
 		goto out;
 	}
 
-	if (param_set_external(param, v, state->mark_saved, true)) {
+	if (param_set_external(param, v, false, false)) {
```

Comment:
Is there a reason to do this different from the main implementation?
```suggestion
	if (param_set_external(param, v, true, true)) {
```

Reply:
No, that's wrong. Fixed.